integration_tests/steps/jira_issues_tests.py

```diff
@@ -122,8 +123,8 @@ def check_matching_trades(context):
 
 @then('there should be no unmatched loads')
 def no_unmatched_loads(context):
-    unmatched = export_unmatched_loads(context.simulation.area)
-    assert unmatched["unmatched_load_count"] == 0
+    unmatched, unmatched_redis = ExportUnmatchedLoads(context.simulation.area).__call__()
```

Comment:
Actually, there is no need for the __call__(), you can just (). This is the reason I suggested the __call__ magic method, to avoid having to even access a new method.

Reply:
OMG! You are right! Will fix this!